jobs/resource_job.go

```diff
@@ -191,6 +197,7 @@ type JobTaskSettings struct {
 	PythonWheelTask *PythonWheelTask `json:"python_wheel_task,omitempty" tf:"group:task_type"`
 	SqlTask         *SqlTask         `json:"sql_task,omitempty" tf:"group:task_type"`
 	DbtTask         *DbtTask         `json:"dbt_task,omitempty" tf:"group:task_type"`
+	RunJobTask      *RunJobTask      `json:"run_job_task,omitempty" tf:"group:task_type"`
```

Comment:
this is not part of the OpenAPI spec (https://docs.databricks.com/api/workspace/jobs/create) - we can't add anything that doesn't have public doc at the moment.

it's a hard requirement

Reply:
The API doc has now supported it. Can this be unblocked?